Mars Mission/Mars Mission/Model/DetailsModel/WeatherDetails.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  WeatherDetails.swift
+//  Mars Mission
+//
+//  Created by Aldrin Bahle Gama on 2020/11/04.
+//
+
+import Foundation
+
+enum WeatherDetails: Int {
+    case sunrise
```

Comment:
Fix indentation here

Reply:
<img width="1272" alt="Screenshot 2020-11-05 at 05 37 43" src="https://user-images.githubusercontent.com/20769103/98194889-14eb0980-1f29-11eb-9efc-ee82427b95c0.png">

fixed thank you.